src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -116,6 +116,7 @@
   private final ExecutorService executorService;
   private final Queue<Future<Void>> hiveUpdateFutures;
   private final Set<String> hivePartitions;
+  private int failedTempFileCloseAttempts = 1;
```

Comment:
It should initialize with 0, or have an entirely different name. Also, it should be added below the last initialized field, before the uninitialized fields (between lines 67 and 68 in this diff).

Reply:
OK.